_posts/2022-11-03-markdown-asciidoc-restructuredtext.md

```diff
@@ -0,0 +1,193 @@
+
+2. Unlike Asciidoctor, [Antora](https://antora.org/) is a true framework for Asciidoc that can store, retrieve and aggregate all Asciidoc content from multiple git repositories. Antora’s page referencing system isn’t coupled to filesystem paths or URLs. You are able cross reference pages across a local machine, a staging environment, and a production environment. To generate a site with Antora, you need the [Antora CLI](https://www.npmjs.com/package/@antora/cli) and [Antora site generator](https://gitlab.com/antora/antora). 
+
+### reStructuredText Framework
+
+If you've noticed the usage "framework" instead of "frameworks" in the above heading, you can guess that there's only one framework for reStructuredText. [Sphinx](https://www.sphinx-doc.org/en/master/index.html),  originally created for Python documentation, takes plain-text files in reStructuredText format and transformes it into HTML, PDF, and any output formats. A few well-known projects that use Sphinx for documentation are [Django](https://docs.djangoproject.com/) and [ReadTheDocs](https://docs.readthedocs.io/en/latest/index.html). 
+
+## Choose the right markup
+
+If you're a developer, learning the syntax and styling for a markup language is relatively easy compared to a programming language. When you learn the syntax of a markup language and can render an HTML page locally, you reach the `hello world` nirvana of documentation. However, the difficulty doesn't necessarily start when you write the first documentation file. I [asked on Twitter](https://twitter.com/DewanAhmed/status/1583465169006448640?s=20&t=kOdJ_vOoZiZ_TJK0vJDK9A) about the choice among Markdown, AsciiDoc, and reStructuredText and this section will summarize the findings.
+
+### Markdown - simple and easy to get started
+
+#### Flavors
+
+Yes, there are [A LOT](https://github.com/commonmark/commonmark-spec/wiki/markdown-flavors) of flavors. Despite all eleventy-zillion flavors, Markdown is loved for developer documentation due to its simplicity. The argument against all the unknown Markdown flavors and incompatibility doesn't stand that much since most projects would choose a common flavor like GFM. 
+
+#### Purpose
+
+For many projects, documentation is about a solid README and some files under the `docs` folder on GitHub. They don't need macros or custom tags within their documentation. Since GitHub renders multiple flavors of Markdown nicely, Markdown is an easy choice for many projects.
+
+#### Syntax
+
+Let's look at a few syntax comparisons to understand how simplicity helps Markdown win many projects. 
+
+Link in reStructuredText:
+
+```
+This is `Dewan's blog <https://www.dewanahmed.com>`_.
+```
```

Comment:
Although this is the form of link we use, it's not intended to be the normal use case - the normal use case is meant to put the URL out-of-band, out of the main reading of the text, since it's a distraction from that text. It's also meant to facilitate the use of a URL more than once.

But I'm not sure how you can explain that, so it may be simpler just to change "Link in reStructuredText" to "Inline link in reStructuredText (there are other ways of doing it)". Or ignore the whole issue 😄